app/assets/javascripts/request-attachments.js

```diff
@@ -0,0 +1,32 @@
+(function($) {
+  $(function () {
+    $('.correspondence .attachments').each(function() {
+      var limit = 3;
+      var link_collapsed_text = 'Show all attachments';
```

Comment:
Just a note that some un-translateable text has slipped in here, not sure if it was always there or has been messed up by moving this into a file?